internal/core/command/init.go

```diff
@@ -237,3 +247,39 @@ func setLoggingTarget() string {
 	}
 	return Configuration.Logging.File
 }
+
+func connectToDatabase() error {
+	var err error
+
+	dbClient, err = newDBClient(Configuration.Databases["Metadata.Primary"].Type)
```

Comment:
Just "Primary" please, not "Metadata.Primary". I should enter an issue to make this a constant.